auth/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "gcloud-aio-auth"
-version = "5.4.1"
+version = "5.4.2"
```

Comment:
I think we might want to keep this in sync with the rest version.. @TheKevJames can you confirm?